src/main/kotlin/step3/validator/InputStringValidator.kt

```diff
@@ -0,0 +1,37 @@
+package step3.validator
+
+object InputStringValidator {
+    /**
+     * 3.3과 같이 Double 값이 입력되는 경우 NumberforamtException을 throw 합니다.
+     * Double로 변경 후에 Int로 변경해줍니다
+     */
+    fun checkValidate(inputString: String?): Int {
```

Comment:
```suggestion
    fun checkValidate(inputString: String): Int {
```
으로 변경한다면 nullable String에 대한 진입을 막고 requireNotNull 검증을 제거할 수 있겠네요.

Reply:
입력에 대해서 `readlnOrNull()`를 사용해서 위와 같이 했는데요, 끝에 orEmpty()를 붙이고 non-null type으로 사용하겠습니다~